client/src/utils/jobsUtil.js

```diff
@@ -1,18 +1,18 @@
 import { getSystemName } from './systems';
 
-const TERMINAL_STATES = [`FINISHED`, `STOPPED`, `FAILED`];
+const TERMINAL_STATES = [`FINISHED`, `CANCELLED`, `FAILED`];
 
 export function isTerminalState(status) {
   return TERMINAL_STATES.includes(status);
 }
 
 // determine if state of job has output
 export function isOutputState(status) {
-  return isTerminalState(status) && status !== 'STOPPED';
+  return isTerminalState(status) && status !== 'CANCELLED';
 }
 
 export function getOutputPath(job) {
-  return `${job.archiveSystemId}/${job.archiveSystemDir}`;
+  return `${job.archiveSystemId}${job.archiveSystemDir}`;
```

Comment:
Yeah I've been testing using the [hello-world](https://github.com/TACC/WMA-Tapis-Templates/tree/main/applications/hello-world) sample application and with that there was an extra `/` appearing since archiveSystemDir already had a leading slash. Maybe I can add a check for a leading slash to determine how to display this?

Reply:
In tapis v2, system definitions had both a root dir and home dir defined. The root dir would act as a top-level limit to the filesystem, so if the root was defined as /home, you could never make a listing above /home. Alongside that, there was a home dir specified relative to the root, like /home/username. If you made a file listing on the system, the listing would return different results depending on whether or not there was a leading slash. e.g. `frontera.home.sal/something` would be relative to the `frontera.home.sal` home dir, whereas `frontera.home.sal//something` would be relative to the `frontera.home.sal` root dir

So, at least in tapis v2, the extra slash is significant, since it spells wether or not the path is relative to the root of the system or the home of the system.

That said, in v3, I'm not sure if that's still a thing. Maybe we can check with cic in the tapis channel?

Reply:
Thanks for that info. I'll confirm with them

Reply:
@rstijerina cic confirmed that there is no `homeDir` anymore only a `rootDir`. In that case, what do you think about the `/` being added back or left as is?

Reply:
Though we should consider the case where `archiveSystemDir` could be without a leading slash. In that case, we will want to make sure there is a slash

Reply:
Sure yeah I can add a check for that